src/sdg/organisaties/management/commands/send_notification_mail.py

```diff
@@ -0,0 +1,104 @@
+
+class Command(BaseCommand):
+    help = f"Send product update notification emails from last ({settings.SDG_MAIL_TEXT_CHANGES_EVERY_DAYS}) days to subscribed users."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--user",
+            metavar=["email", "username"],
+            help="Send to email to one specific user instead based on 'email' or 'username'.",
+        )
+
+    def get_user_by_email(self, user):
+        return (
+            User.objects.filter(email=user, roles__ontvangt_mail=True)
+            .order_by("email")
+            .distinct("email")
+        )
+
+    def get_user_by_username(self, user):
+        user_qs = (
+            User.objects.filter(roles__ontvangt_mail=True)
+            .order_by("email")
+            .distinct("email")
+        )
+        if user_qs:
+            for search_term in user.split():
+                user_qs = user_qs.filter(
+                    Q(first_name__iexact=search_term) | Q(last_name__iexact=search_term)
+                )
+        return user_qs
+
+    def handle(self, user, **options):
+        x_days_ago = now() - relativedelta(
+            days=settings.SDG_MAIL_TEXT_CHANGES_EVERY_DAYS
+        )
+        product_versies = (
+            ProductVersie.objects.filter(
+                product__referentie_product=None,
+                gewijzigd_op__gte=x_days_ago,
+            )
+            .published()
+            .order_by("-gewijzigd_op")
+        )
+        if product_versies:
+            if user:
+                user_qs = self.get_user_by_email(user)
+                if not user_qs:
+                    user_qs = self.get_user_by_username(user)
+            else:
+                user_qs = (
+                    User.objects.filter(roles__ontvangt_mail=True)
+                    .order_by("email")
+                    .distinct("email")
+                )
+
+            if user_qs:
+                organisation_name = org_type_cfg().organisation_name
+
+                for user in user_qs:
+                    mail_ctx = {
+                        "user_full_name": user.get_full_name(),
+                        "product_versions": product_versies,
+                        "org_name": organisation_name,
+                    }
+
+                    html_message = render_to_string(
+                        "organisaties/notificaties/email/notificatie.html",
+                        context=mail_ctx,
+                    )
+
+                    send_mail(
+                        "Aanpassing in tekst SDG-invoervoorziening",
+                        strip_tags(html_message),
+                        settings.DEFAULT_FROM_EMAIL,
+                        [user.email],
+                        html_message=html_message,
+                    )
+
+                self.stdout.write(
+                    self.style.SUCCESS(
+                        f"Successfully send emails to {len(user_qs)} user(s)"
+                    )
+                )
```

Comment:
Can you add some unhappy-flow messages, like this:

```suggestion
                )
            else:
                    self.style.SUCCESS(
                        "No eligable users found."
                    )
        else:
                self.style.SUCCESS(
                   f"No changed products found in the past {x_days_ago} days."
               )

```